lib/Robo/Plugin/Commands/RepairCommands.php

```diff
@@ -0,0 +1,159 @@
+ *
+ * The interactive user interfaces in modified source and object code versions
+ * of this program must display Appropriate Legal Notices, as required under
+ * Section 5 of the GNU Affero General Public License version 3.
+ *
+ * In accordance with Section 7(b) of the GNU Affero General Public License version 3,
+ * these Appropriate Legal Notices must retain the display of the "Powered by
+ * SugarCRM" logo and "Supercharged by SuiteCRM" logo. If the display of the logos is not
+ * reasonably feasible for technical reasons, the Appropriate Legal Notices must
+ * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
+ */
+
+namespace SuiteCRM\Robo\Plugin\Commands;
+
+/**
+ * Class RepairCommands
+ *
+ * @category RoboTasks
+ * @package  SuiteCRM\Robo\Plugin\Commands
+ * @author   Jose C. Massón <jose AT gcoop DOT coop>
+ * @license  GNU GPLv3
+ * @link     RepairCommands
+ */
+
+
+class RepairCommands extends \Robo\Tasks
+{
+    /**
+     * Repair database - Synchronize database with vardefs
+     * @param array $opts optional command line arguments
+     * execute - Set if you want that the command executes the SQL or not.
+     * @throws \RuntimeException
+     */
+    public function repairDatabase(array $opts = ['execute' => 'yes'])
+    {
+        global  $beanFiles;
+        $this->say('Repairing database...');
+        $db = \DBManagerFactory::getInstance();
+        $queries = [];
+        \VardefManager::clearVardef();
+        $execute = false;
+
+        if ($opts['execute'] === 'yes') {
+            $execute = true;
+        }
+
+        foreach ($beanFiles as $bean_name => $file) {
+            if (!file_exists($file)) {
+                continue;
+            }
+
+            require_once $file;
+            $GLOBALS['reload_vardefs'] = true;
+            $focus = new $bean_name();
+
+            if (isset($focus->disable_vardefs) && $focus->disable_vardefs == false && isset($focus->module_dir)) {
+                include 'modules/'.$focus->module_dir.'/vardefs.php';
+                $sql = $db->repairTable($focus, $execute);
+
+                if (!empty($sql)) {
+                    $queries[] = $sql;
+                }
+            }
+        }
+
+        $total = count($queries);
+
+        if ($execute) {
+            $this->say("Database synchronized with vardefs!");
+            $this->say("Executed queries: {$total}");
+            return;
+        }
+
+        $this->say("Execute the following queries {$total} in order to get database synchronized with vardefs");
+        array_map('print_r', $queries);
+    }
+
+
+
+    /**
+     * Rebuild Extensions - This Robo task executes rebuildExtensions()
+     * @param array $opts optional command line arguments
+     * show-output - Set if you want to see the rebuildExtensions() output.
+     * @throws \RuntimeException
+     */
+    public function repairRebuildExtensions(array $opts = ['show-output' => 'no'])
```

Comment:
This should also be converted to a boolean.

```suggestion
    public function repairRebuildExtensions(array $opts = ['show-output' => false])
```

And then we can just use `$opts['show-output']` in the function.